lib/matplotlib/collections.py

```diff
@@ -1262,62 +1262,60 @@ def __init__(self,
                  **kwargs
                  ):
         """
-        *positions*
-            a sequence of numerical values or a 1D numpy array.  Can be None
-
-        *orientation* [ 'horizontal' | 'vertical' | None ]
-            defaults to 'horizontal' if not specified or None
+        Parameters
+        ----------
+        positions : 1D array-like object
+            Each value is an event.
 
-        *lineoffset*
-            a single numerical value, corresponding to the offset of the center
-            of the markers from the origin
+        orientation : {None, 'horizontal', 'vertical'}, optional
+            The orientation of the **collection** (the event bars are along
+            the orthogonal direction). Defaults to 'horizontal' if not
+            specified or None.
 
-        *linelength*
-            a single numerical value, corresponding to the total height of the
-            marker (i.e. the marker stretches from lineoffset+linelength/2 to
-            lineoffset-linelength/2).  Defaults to 1
+        lineoffset : scalar, optional, default: 0
+            The offset of the center of the markers from the origin, in the
+            direction orthogonal to *orientation*.
 
-        *linewidth*
-            a single numerical value
+        linelength : scalar, optional, default: 1
+            The total height of the marker (i.e. the marker stretches from
+            ``lineoffset - linelength/2`` to ``lineoffset + linelength/2``).
 
-        *color*
-            must be a sequence of RGBA tuples (e.g., arbitrary color
-            strings, etc, not allowed).
+        linewidth : scalar or None, optional, default: None
+            If it is None, defaults to its rcParams setting, in sequence form.
 
-        *linestyle* [ 'solid' | 'dashed' | 'dashdot' | 'dotted' ]
+        color : color, sequence of colors or None, optional, default: None
+            If it is None, defaults to its rcParams setting, in sequence form.
 
-        *antialiased*
-            1 or 2
+        linestyle : str or tuple, optional, default: 'solid'
+            Valid strings are ['solid', 'dashed', 'dashdot', 'dotted',
+            '-', '--', '-.', ':']. Dash tuples should be of the form::
 
-        If *linewidth*, *color*, or *antialiased* is None, they
-        default to their rcParams setting, in sequence form.
+                (offset, onoffseq),
 
-        *norm*
-            None (optional for :class:`matplotlib.cm.ScalarMappable`)
-        *cmap*
-            None (optional for :class:`matplotlib.cm.ScalarMappable`)
+            where *onoffseq* is an even length tuple of on and off ink
+            in points.
 
-        *pickradius* is the tolerance for mouse clicks picking a line.
-        The default is 5 pt.
+        antialiased : {None, 1, 2}, optional
+            If it is None, defaults to its rcParams setting, in sequence form.
```

Comment:
TBH I have no idea of what this parameter is doing except that it has to do with AA. So I tried to avoid changing the original sentence as much as possible.

From a very quick look at other docstrings in `colllections`, a sequence form is mentioned very often for the parameters `antialiased(s)`. However, it is not totally consistent with this one here, as most of them state that is has to be boolean or sequence of boolean (or 0 and 1 values)… I'll try to have a deeper look today.